charts/latest/csi-driver-smb/values.yaml

```diff
@@ -147,6 +147,7 @@ windows:
     dsName: csi-proxy-win # daemonset name
     tolerations: {}
     affinity: {}
+    username: "NT AUTHORITY\\SYSTEM"
```

Comment:
what's this change?

Reply:
Fix: CSI proxy for windows defaults did not match documented defaults